features/logging/collector.feature

```diff
@@ -1,29 +1,55 @@
 @clusterlogging
 Feature: collector related tests
 
+  # @auther qitang@redhat.com
+  # @case_id OCP-25767
+  @admin
+  @destructive
+  @commonlogging
+  Scenario: All nodes logs are sent to Elasticsearch
+    Given the master version == "4.1"
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-logging" project
+    Given evaluation of `cluster_logging('instance').fluentd_ready_pod_names` is stored in the :collector_pods clipboard
+
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
+    And the expression should be true> cb.index_data['docs.count'] > "0"
+    And I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | _search?pretty&size=0' -H 'Content-Type: application/json' -d'{"aggs" : {"exists_field_kubernetes" : {"filter": {"exists": {"field":"kubernetes"}},"aggs" : {"distinct_hostname_name" : {"terms" : {"field" : "pipeline_metadata.collector.ipaddr4"}}}}}} |
```

Comment:
I think we should rename the distinct_hostname_name  to distinct_fluentd_ip or distinct_node_ip

Reply:
OK, I'll change the name